ui/components/app/confirm/info/row/url.tsx

```diff
@@ -27,17 +27,23 @@ export type ConfirmInfoRowUrlProps = {
 export const ConfirmInfoRowUrl = ({ url }: ConfirmInfoRowUrlProps) => {
   let urlObject;
   const [isModalOpen, setIsModalOpen] = useState(false);
-  const openModal = () => setIsModalOpen(true);
-  const closeModal = () => setIsModalOpen(false);
+  const handleOpenModal = useCallback(
+    () => setIsModalOpen(true),
+    [setIsModalOpen],
+  );
+  const handleCloseModal = useCallback(
+    () => setIsModalOpen(false),
+    [setIsModalOpen],
+  );
 
   if (isSnapId(url)) {
     return (
       <>
-        <SnapAuthorshipPill snapId={url} onClick={openModal} />
+        <SnapAuthorshipPill snapId={url} onClick={handleOpenModal} />
```

Comment:
Minor, sorry for the confusion. In these examples it could be `handlePillClick` or `handleModalClose`.

Reply:
[2b94344](https://github.com/MetaMask/metamask-extension/pull/26881/commits/2b94344f14c49acc4f9efc7138ca3983309bf2b5)